cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -129,52 +150,54 @@ export default {
       const _filter = (id) => {
         return !contents[id].type.some((t) => filterBy.includes(t));
       };
 
-      const _createCard = (id) => {
-        return { content: contents[id], flipped: false };
-      };
+      this.credentials = Object.keys(contents)
+        .filter(_filter)
+        .map((id) => contents[id]);
 
-      this.cards = Object.keys(contents).filter(_filter).map(_createCard);
+      console.debug(`showing ${this.credentials.length} credentials`);
 
-      console.log(`showing ${this.cards.length} credentials`);
-
-      // Reading the values of the credentials and mapping it to the credential display data schemas
-      this.cjson.forEach((obj) => {
-        let flattened = {};
-        for (let credentialKey in this.cards) {
-          this.flatten(this.cards[credentialKey].content, flattened);
-          for (let credentialContent in flattened) {
-            if (
-              obj.credentialSubject.hasOwnProperty(credentialContent) &&
-              obj.schema === this.credDisplayName(this.cards[credentialKey].content)
-            ) {
-              obj.credentialSubject[credentialContent] = flattened[credentialContent];
+      Object.values(this.credentials).map((credential) => {
+        const currentCredentialType = this.getCredentialType(credential);
+        Object.entries(this.supportedCredentials).map(
+          ([supportedCredentialType, supportedCredential]) => {
+            if (supportedCredentialType === currentCredentialType) {
+              const processedCredential = this.getCredentialDisplayData(
+                credential,
+                supportedCredential
+              );
+              this.processedCredentials.push({ ...processedCredential, category: 'manifested' });
             }
           }
+        );
+        if (!this.supportedCredentials.hasOwnProperty(currentCredentialType)) {
+          this.processedCredentials.push({ ...credential, category: 'generic' });
         }
       });
+      console.debug(`showing ${this.processedCredentials.length} credentials`);
+    },
+    getCredentialType: function (vc) {
+      return getCredentialType(vc.type);
     },
     credDisplayName: function (vc) {
       return vc.name ? vc.name : getCredentialType(vc.type);
     },
-    loadImg(imgPath) {
-      return images('./' + imgPath);
+    getCredentialDisplayData: function (vc, manifestCredential) {
+      return getCredentialDisplayData(vc, manifestCredential);
     },
-    flatten: function (json, flattened) {
-      for (let key in json) {
-        if (json.hasOwnProperty(key)) {
-          if (json[key] instanceof Object && json[key] != '') {
-            this.flatten(json[key], flattened, key);
-          } else {
-            flattened[key] = json[key];
-          }
-        }
+    replacer: function (key, value) {
+      if (typeof value === 'function') {
+        return value.toString();
       }
+      return value;
+    },
```

Comment:
where is this user?

Reply:
this was supposed to be deleted, so removed

Reply:
`replacer` is actually not used anywhere, we should get rid of it @talwinder50 @sudeshrshetty